extensions/panache/hibernate-reactive-panache/runtime/src/main/java/io/quarkus/hibernate/reactive/panache/Panache.java

```diff
@@ -0,0 +1,85 @@
+package io.quarkus.hibernate.reactive.panache;
+
+import java.util.Map;
+import java.util.function.Supplier;
+
+import javax.persistence.EntityManager;
+import javax.transaction.TransactionManager;
+
+import org.hibernate.reactive.mutiny.Mutiny;
+
+import io.quarkus.hibernate.reactive.panache.runtime.JpaOperations;
+import io.quarkus.panache.common.Parameters;
+import io.smallrye.mutiny.Uni;
+
+/**
+ * Utility class for Panache.
+ *
+ * @author Stéphane Épardaud
+ */
+public class Panache {
+
+    /**
+     * Returns the current {@link EntityManager}
```

Comment:
It's not an `EntityManager` .. in this case since it's public API I'd definitely fix it.

Reply:
Yes, I've fixed it.